### DISCUSSION TITLE
Does pixi skip applying filter for the part which is not visible?

### DISCUSSION THREAD

**Discussion by a-c-sreedhar-reddy** [pixijs/pixijs#7646] [Q&A] [answered]
### Question
Hi, in the following case half of the image is out of the canvas. 
In this case would pixi.js skip applying the filters on the part outside the canvas?

![image](https://user-images.githubusercontent.com/16081083/127274092-62a07e45-ef90-4512-acaf-c55ea35a020c.png)

**Comment by ivanpopelyshev** [comment, marked as answer]
Filter is applied on rect inside screen. If its not enough for your filter, you can turn it off `filter.autoFit = false`. Usually when you are doing some normalized coord manipulation. 

Alternative: if its just one sprite, and not multipass filter, use mesh-shader https://pixijs.io/examples/#/mesh-and-shaders/shadertoy-mesh.js

**Reply by a-c-sreedhar-reddy** [reply]
Thanks @ivanpopelyshev disabling autoFit  helped.
